packages/Card/src/components/Title/Title.styles.js

```diff
@@ -8,4 +8,5 @@ export const titleStyles = styled.h1`
   ${stylers.fontSize(0)};
   margin: 0;
   ${stylers.truncateText};
+  padding: ${tokens.spaceSm};
```

Comment:
```suggestion
  padding: 0 ${tokens.spaceSm};
```

Reply:
I don't think the intention was for this text to be indented.

Reply:
Actually, I think perhaps `margin` would be more appropriate than `padding` for this and the other text spacing?  Margins overlap, so if you have 4px bottom margin and 8px top margin on the next item, the actual spacing will be 8px.  If you do the same with padding, the actual spacing will be 12px.